Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.yml

```diff
@@ -0,0 +1,2251 @@
+          type: String
+        - contextPath: Bitbucket.Commit.author.user.type
+          description: 'The type of the user of the author.'
+          type: String
+        - contextPath: Bitbucket.Commit.author.user.uuid
+          description: 'The user unique key of the author.'
+          type: String
+        - contextPath: Bitbucket.Commit.author.user.account_id
+          description: 'The user account id of the author.'
+          type: String
+        - contextPath: Bitbucket.Commit.author.user.nickname
+          description: 'The user nickname of the author.'
+          type: String
+        - contextPath: Bitbucket.Commit.message
+          description: 'The commit message'
+          type: String
+        - contextPath: Bitbucket.Commit.summary.type
+          description: 'The type of the summary.'
+          type: String
+        - contextPath: Bitbucket.Commit.summary.raw
+          description: 'The raw summary of the commit.'
+          type: String
+        - contextPath: Bitbucket.Commit.summary.markup
+          description: 'The text styling type, such as markdown.'
+          type: String
+        - contextPath: Bitbucket.Commit.summary.html
+          description: 'The summary in html format.'
+          type: String
+        - contextPath: Bitbucket.Commit.links
+          description: 'Links with information about the commit.'
+          type: String
+        - contextPath: Bitbucket.Commit.parents.type
+          description: 'The type of the commit parents.'
+          type: String
+        - contextPath: Bitbucket.Commit.parents.hash
+          description: 'The hash of the commit parents'
+          type: String
+        - contextPath: Bitbucket.Commit.parents.links
+          description: 'Links with information about the parents.'
+          type: String
+        - contextPath: Bitbucket.Commit.repository.type
+          description: 'The type of the repository.'
+          type: String
+        - contextPath: Bitbucket.Commit.repository.full_name
+          description: 'The full name of the repository.'
+          type: String
+        - contextPath: Bitbucket.Commit.repository.links.self.href
+          description: 'Links with information about the repository.'
+          type: String
+        - contextPath: Bitbucket.Commit.repository.name
+          description: 'The name of the repository.'
+          type: String
+        - contextPath: Bitbucket.Commit.repository.uuid
+          description: 'The unique id of the repository'
+          type: String
+    - name: bitbucket-file-delete
+      arguments:
+        - name: repo
+          required: false
+          description: The repository name or slug.
+          isArray: false
+        - name: message
+          required: true
+          description: Commit a message with the file.
+          isArray: false
+        - name: branch
+          required: true
+          description: This branch will be associated with the commited file.
```

Comment:
Is this the right description?

Reply:
No. It is a copy & paste error. I fixed it